sdk/tables/azure-data-tables/README.md

```diff
@@ -60,6 +60,7 @@ The `credential` parameter may be provided in a number of different forms, depen
 * Shared Key
 * Connection String
 * Shared Access Signature Token
+* TokenCredential(AAD)
```

Comment:
I think this is for Storage only - I don't think it's supported by Cosmos, so we should make a note of that.

Reply:
Added.